src/patternfly/components/Popover/docs/code.md

```diff
@@ -21,10 +21,10 @@ A popover is used to provide contextual information for another component on cli
 | `.pf-c-popover` | `<div>` |  Creates a popover. Always use it with a modifier class that positions the popover relative to the element that triggered it. **Required** |
 | `.pf-c-popover__arrow` | `<div>` |  Creates an arrow pointing towards the element the popover describes. **Required** |
 | `.pf-c-popover__content` | `<div>` |  Creates the body of the popover. **Required** |
-| `.pf-c-popover__close` | `<div>` |  Contains and positions the close icon in the top-right corner of the popover. **Required** |
-| `.pf-c-popover__header` | `<header>` |  The header text area of the popover. |
-| `.pf-c-popover__header-title` | `<h1>`,`<h2>`,`<h3>`,`<h4>`,`<h5>`,`<h6>` |  The actual popover title. |
+| `.pf-c-button` | `<button>` |  Positions the close icon in the top-right corner of the popover. **Required** |
+| `.pf-c-title` | `<h1>`,`<h2>`,`<h3>`,`<h4>`,`<h5>`,`<h6>` |  Initiates a title. Always use it with a modifier class.|
```

Comment:
can we add here "see title component for more info" in case it's not obvious what the modifiers would be.